hawk/lib/crm_history.rb

```diff
@@ -9,11 +9,12 @@
 # - Run new command
 
 module CrmHistory
-  @pidfile = "#{Rails.root}/tmp/pids/crm_history.pid"
-  @cmdfile = "#{Rails.root}/tmp/pids/crm_history.cmd"
-  @outfile = "#{Rails.root}/tmp/pids/crm_history.stdout"
-  @errfile = "#{Rails.root}/tmp/pids/crm_history.stderr"
-  @exitfile = "#{Rails.root}/tmp/pids/crm_history.exit"
+  path = Rails.root.join('tmp', 'pids')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.